src/controllers/orders.controllers.ts

```diff
@@ -0,0 +1,45 @@
+import {Request, Response} from 'express'; 
+import {Types} from 'mongoose'; 
+const Order = require("../models/Order.ts");
+
+interface CreateQuery {
+    numDiapers: number; 
+}
+
+interface GetQuery {
+    orderId?: Types.ObjectId, 
+    partnerId?: Types.ObjectId, 
+    status?: string;
+}
+
+const createOrder = async (req:Request<unknown, unknown, unknown, CreateQuery>, res:Response) => {
+    try {
+        const { query } = req; 
```

Comment:
For create order and get order, we should be getting fields (eg numDiapers) from req.body. Feel free to look at past years projects to see how this is generally done (https://github.com/ChangePlusPlusVandy/hfc-backend/blob/main/controllers/assessments.controller.js).  Doing it this way will make developing the frontend a lot easier!!